src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -7,6 +7,33 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Removed the following package references:
+
+  * `Google.Protobuf`
+  * `Grpc`
+  * `Grpc.Net.Client`
+  * `Grpc.Tools`
```

Comment:
nit: You don't really need to mention `Grpc.Tools`. It was a `PrivateAssets` thing so it never flowed to users anyway.